### PR TITLE
Add contribute.json file.

### PR DESCRIPTION
**Pull request by Osmose**
Add a contribute.json file à la http://contribute.paas.allizom.org/.

### CODE REVIEW

**Comment by Osmose** [comment]
r?

**Comment by Osmose** [comment]
@erikrose r?

File: contribute.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+    "name": "DXR",
+    "description": "An intelligent source code browser.",
```
Comment (erikrose) [inline contribute.json]: "Mozilla's code search and navigation tool, aimed at making sense of large projects like Firefox"

File: contribute.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    "repository": {
+        "url": "https://github.com/mozilla/dxr",
+        "license": "MIT",
+        "tests": "https://ci.mozilla.org/job/dxr/"
+    },
+    "participate": {
+        "home": "https://dxr.readthedocs.org/en/latest/community.html",
+        "docs": "https://dxr.readthedocs.org/en/latest/development.html",
```
Comment (erikrose) [inline contribute.json]: Should this be https://dxr.readthedocs.org/en/latest/index.html, or is the semantic "developer docs"?

File: contribute.json [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        "irc": "irc://irc.mozilla.org/#static",
+        "irc-contacts": ["ErikRose"]
+    },
+    "bugs": {
+        "list": "https://bugzilla.mozilla.org/buglist.cgi?order=Importance&resolution=---&query_format=advanced&component=DXR&product=Webtools",
+        "report": "https://bugzilla.mozilla.org/enter_bug.cgi?product=Webtools&component=DXR",
+        "mentored": "https://bugzilla.mozilla.org/buglist.cgi?quicksearch=whiteboard%3Aeasy%20component%3ADXR&list_id=6127603"
```
Comment (erikrose) [inline contribute.json]: Nice mapping—to the easy bugs!

File: contribute.json [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    },
+    "urls": {
+        "prod": "https://dxr.mozilla.org/",
+        "stage": "https://dxr.allizom.org/"
+    },
+    "keywords": [
+        "flask",
+        "jquery",
+        "python",
+        "javascript"
```
Comment (erikrose) [inline contribute.json]: "static analysis",

**Comment by erikrose** [comment]
r+ with comments

File: contribute.json (hunk 2 above)
Comment (Osmose) [inline contribute.json]: Air Mozilla's points to the part of their docs for setting up a developer instance, so I've been going with that. I'm not sure the semantics are super-clear.

**Pull request merged**